runtime/interpreter/statictype_test.go

```diff
@@ -972,3 +973,487 @@ func TestPrimitiveStaticTypeCount(t *testing.T) {
+			semaType:   sema.FixedPointType,
+			staticType: PrimitiveStaticTypeFixedPoint,
+		},
+		{
+			name:       "SignedFixedPoint",
+			semaType:   sema.SignedFixedPointType,
+			staticType: PrimitiveStaticTypeSignedFixedPoint,
+		},
+
+		{
+			name:       "Int",
+			semaType:   sema.IntType,
+			staticType: PrimitiveStaticTypeInt,
+		},
+		{
+			name:       "Int8",
+			semaType:   sema.Int8Type,
+			staticType: PrimitiveStaticTypeInt8,
+		},
+		{
+			name:       "Int16",
+			semaType:   sema.Int16Type,
+			staticType: PrimitiveStaticTypeInt16,
+		},
+		{
+			name:       "Int32",
+			semaType:   sema.Int32Type,
+			staticType: PrimitiveStaticTypeInt32,
+		},
+		{
+			name:       "Int64",
+			semaType:   sema.Int64Type,
+			staticType: PrimitiveStaticTypeInt64,
+		},
+		{
+			name:       "Int128",
+			semaType:   sema.Int128Type,
+			staticType: PrimitiveStaticTypeInt128,
+		},
+		{
+			name:       "Int256",
+			semaType:   sema.Int256Type,
+			staticType: PrimitiveStaticTypeInt256,
+		},
+
+		{
+			name:       "UInt",
+			semaType:   sema.UIntType,
+			staticType: PrimitiveStaticTypeUInt,
+		},
+		{
+			name:       "UInt8",
+			semaType:   sema.UInt8Type,
+			staticType: PrimitiveStaticTypeUInt8,
+		},
+		{
+			name:       "UInt16",
+			semaType:   sema.UInt16Type,
+			staticType: PrimitiveStaticTypeUInt16,
+		},
+		{
+			name:       "UInt32",
+			semaType:   sema.UInt32Type,
+			staticType: PrimitiveStaticTypeUInt32,
+		},
+		{
+			name:       "UInt64",
+			semaType:   sema.UInt64Type,
+			staticType: PrimitiveStaticTypeUInt64,
+		},
+		{
+			name:       "UInt128",
+			semaType:   sema.UInt128Type,
+			staticType: PrimitiveStaticTypeUInt128,
+		},
+		{
+			name:       "UInt256",
+			semaType:   sema.UInt256Type,
+			staticType: PrimitiveStaticTypeUInt256,
+		},
+
+		{
+			name:       "Word8",
+			semaType:   sema.Word8Type,
+			staticType: PrimitiveStaticTypeWord8,
+		},
+		{
+			name:       "Word16",
+			semaType:   sema.Word16Type,
+			staticType: PrimitiveStaticTypeWord16,
+		},
+		{
+			name:       "Word32",
+			semaType:   sema.Word32Type,
+			staticType: PrimitiveStaticTypeWord32,
+		},
+		{
+			name:       "Word64",
+			semaType:   sema.Word64Type,
+			staticType: PrimitiveStaticTypeWord64,
+		},
+
+		{
+			name:       "Fix64",
+			semaType:   sema.Fix64Type,
+			staticType: PrimitiveStaticTypeFix64,
+		},
+
+		{
+			name:       "UFix64",
+			semaType:   sema.UFix64Type,
+			staticType: PrimitiveStaticTypeUFix64,
+		},
+
+		{
+			name:       "Path",
+			semaType:   sema.PathType,
+			staticType: PrimitiveStaticTypePath,
+		},
+		{
+			name:       "Capability",
+			semaType:   &sema.CapabilityType{},
+			staticType: PrimitiveStaticTypeCapability,
+		},
+		{
+			name:       "StoragePath",
+			semaType:   sema.StoragePathType,
+			staticType: PrimitiveStaticTypeStoragePath,
+		},
+		{
+			name:       "CapabilityPath",
+			semaType:   sema.CapabilityPathType,
+			staticType: PrimitiveStaticTypeCapabilityPath,
+		},
```

Comment:
This one is redundant. "Unparameterized Capability" test below is same as this one

Reply:
This test is for the `CapabilityPath` type, not the `Capability` type.

Reply:
ah right! my bad! My eyes have been delusional